doc/controls-styles.md

```diff
@@ -18,6 +18,8 @@ Control|Style Key|IsDefaultStyle*|
 -|-|-
```

Comment:
Yeah :( I put this in Toolkit instead of Uno.Material in Themes to save time but I think it makes more sense to keep them here in Toolkit. The MaterialAppBarButtonStyle's purpose is to style the NavigationBar/CommandBar buttons specifically.